Patches/VXL/vidl_ffmpeg_ostream_v56.hxx

```diff
@@ -0,0 +1,606 @@
+// This is core/vidl/vidl_ffmpeg_ostream_v56.hxx
```

Comment:
Is this a new file or patched from VXL source? If it's patched, it would be ideal to add in 2 commits so we can see the actual diff